src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -100,6 +104,14 @@ private void UpdateParameterMetadata(ModelMetadata metadata, IBinderMetadata bin
                     modelBindingContext.IsModelSet)
                 {
                     arguments[parameter.PropertyName] = modelBindingContext.Model;
+                    var validationContext = new ModelValidationContext(
+                    _modelMetadataProvider,
```

Comment:
I think otherwise.

if someone needs this service they should DI it, we don't want to pass around services as arguments or as properties on a context